internal/services/trafficmanager/azure_endpoint_resource.go

```diff
@@ -170,10 +177,13 @@ func resourceAzureEndpointCreate(d *pluginsdk.ResourceData, meta interface{}) er
 		status = endpoints.EndpointStatusDisabled
 	}
 
+	alwaysServe := endpoints.AlwaysServe(d.Get("always_serve").(string))
+
 	params := endpoints.Endpoint{
 		Name: utils.String(id.EndpointName),
 		Type: utils.String(fmt.Sprintf("Microsoft.Network/trafficManagerProfiles/%s", endpoints.EndpointTypeAzureEndpoints)),
 		Properties: &endpoints.EndpointProperties{
+			AlwaysServe:      &alwaysServe,
```

Comment:
this can be changed to
```suggestion
			AlwaysServe:      pointer.To(endpoints.AlwaysServe(d.Get("always_serve").(string))),
```